### PR TITLE
Update benchmark page proposal

### PR DESCRIPTION
**Pull request by iamleonie**
### What's being changed:

Major re-structuring and re-organizing of contents of Benchmark sites.

### Type of change:

- [x] **Documentation** updates (non-breaking change to fix/update documentation)
- [ ] **Website** updates (non-breaking change to update main page, company pages, pricing, etc)
- [ ] **Content** updates – **blog**, **podcast** (non-breaking change to add/update content)
- [ ] **Bug fix** (non-breaking change to fixes an issue with the site)
- [ ] **Feature** or **enhancements** (non-breaking change to add functionality)

### How Has This Been Tested?

- [x] **Github action** – automated build completed without errors
- [x] **Local build** - the site works as expected when running `yarn start`

> note, you can run `yarn verify-links` to test site links locally

### CODE REVIEW

File: developers/weaviate/benchmarks/ann.md [hunk 1]
```diff
@@ -233,7 +183,74 @@ import AnnGist960 from '/_includes/ann-gist-960.mdx';
...
+The imports were performed using Weaviate's Python clients. The concurrent
+(multi-threaded) queries were measured using Go. Each language may have a
+slightly different performance, and you may experience different results if you
+send your queries using another language. For the maximum throughput, we
+recommend using the [Go](/developers/weaviate/client-libraries/go.md) or
+[Java](/developers/weaviate/client-libraries/java.md) clients.
+
+The complete import and test scripts are available [here](https://github.com/weaviate/weaviate-benchmarking).
+
+## Learn more & Benchmark FAQ
```
Comment (daveatweaviate) [inline developers/weaviate/benchmarks/ann.md:248]: Consider:

```
## Benchmark FAQ
```

File: developers/weaviate/benchmarks/ann.md [hunk 2]
```diff
@@ -233,7 +183,74 @@ import AnnGist960 from '/_includes/ann-gist-960.mdx';
...
+
+### How can I get the most performance for my use case?
+If your use case is similar to one of the showcased datasets this benchmark, you can start with the recommended configurations for the HNSW parameters.
```
Comment (daveatweaviate) [inline developers/weaviate/benchmarks/ann.md:251]: Consider:
If your use case is similar to one of the benchmark tests, use the recommended HNSW parameter configurations to start tuning.

File: developers/weaviate/benchmarks/ann.md [hunk 3]
```diff
@@ -233,7 +183,74 @@ import AnnGist960 from '/_includes/ann-gist-960.mdx';
...
+
+For more instructions on how to tune your configuration for best performance, you can refer to our [HNSW Configuration Tips](https://weaviate.io/developers/weaviate/config-refs/schema/vector-index#hnsw-configuration-tips).
```
Comment (daveatweaviate) [inline developers/weaviate/benchmarks/ann.md:253]: Consider:
best performance, see [HNSW Configuration Tips]

**Review by daveatweaviate — COMMENTED** [review]
Thanks @iamleonie  - some cool experiments!

I think a bunch of these comments refer to existing text and not the updates, hopefully that's ok. 

I had one question, normally benchmarks report average scores over some number of runs. I didn't see that here. Maybe we could add a note somewhere about how many runs there were. Or, if there's only one run because of cost, let's add a note about that instead. 

Let me know if you have any questions!

File: developers/weaviate/benchmarks/ann.md [hunk 4]
```diff
@@ -1,47 +1,66 @@
...
+# tags: ['Weaviate', 'performance', 'benchmarks', 'ANN benchmarks', 'vector database benchmarks']
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-# About this benchmark
+# ANN Benchmark
 
 This benchmark is designed to measure and illustrate Weaviate's ANN performance for a range of real-life use cases.
```
[the remark this answers is not in the document]
Comment (iamleonie) [reply developers/weaviate/benchmarks/ann.md:13]: Fixee, but ANN in this context stands for Approximate Nearest Neighbor :)

File: developers/weaviate/benchmarks/ann.md [hunk 5]
```diff
@@ -1,47 +1,66 @@
...
 - **The overall performance** – Review the [benchmark result section below](#results) to draw conclusions about what to expect from Weaviate in a production setting.
 - **Expectation for your use case** – Find the dataset closest to your production use case, and estimate Weaviate's expected performance for your use case.
-- **Fine Tuning** – If you don't get the results you expect. Find the optimal combinations of the config parameters (efConstruction, maxConnections and ef) to achieve the best results for your production configuration.
+- **Fine Tuning** – If you don't get the results you expect. Find the optimal combinations of the configuration parameters (`efConstruction`, `maxConnections` and `ef`) to achieve the best results for your production configuration. (See [HNSW Configuration Tips](https://weaviate.io/developers/weaviate/config-refs/schema/vector-index#hnsw-configuration-tips))
+
+:::warning TODO
+Add some more info related to our philosophy / [DuckDB blog post](https://motherduck.com/blog/perf-is-not-enough/)
+:::
 
 ## What is being measured?
```
[the remark this answers is not in the document]
Comment (iamleonie) [reply developers/weaviate/benchmarks/ann.md:30]: Decided to go with "What metrics are measured"

File: developers/weaviate/benchmarks/ann.md [hunk 6]
```diff
@@ -54,100 +73,32 @@ users would also experience. In particular, these means:
...
-Based on your throughput requirements, it is very likely that you will run Weaviate
-on a considerably smaller or larger machine in production.
-
-In [this section below](#what-happens-if-i-run-with-fewer-or-more-cpu-cores-than-on-the-example-test-machine)
-we have outlined what you should expect when altering the configuration or
-setup parameters.
+## Benchmark Results
 
-### Experiment Setup
+**Last updated: MM 20YY with Version**
```
[the remark this answers is not in the document]
Comment (iamleonie) [reply developers/weaviate/benchmarks/ann.md:78]: No, this is just shown weirdly due to reorganization.

File: developers/weaviate/benchmarks/ann.md [hunk 7]
```diff
@@ -54,100 +73,32 @@ users would also experience. In particular, these means:
...
-(multi-threaded) queries were measured using Go. Each language may have a
-slightly different performance, and you may experience different results if you
-send your queries using another language. For the maximum throughput, we
-recommend using the [Go](/developers/weaviate/client-libraries/go.md) or
-[Java](/developers/weaviate/client-libraries/java.md) clients.
 
-The complete import and test scripts are available [here](https://github.com/weaviate/weaviate-benchmarking).
+| **Dataset** | **Number of Objects** | **Vector Dimensions** | **Distance metric** | **Use case** |
+| --- | --- | --- | --- | --- |
+| [SIFT1M](tbd) | 1 M | 128 | L2 | This dataset reflects a common use case with a small number of objects. |
```
[the remark this answers is not in the document]
Comment (iamleonie) [reply developers/weaviate/benchmarks/ann.md:85]: I think this always refers to search, so I don't think we need to specify the use case.

File: developers/weaviate/benchmarks/ann.md [hunk 8]
```diff
@@ -233,7 +183,74 @@ import AnnGist960 from '/_includes/ann-gist-960.mdx';
...
+
+* It is large enough to show that Weaviate is a highly-concurrent vector search
+  engine and scales well while running thousands of searches across multiple threads.
+* It is small enough to represent a typical production case without inducing
+  high costs.
+
+Based on your throughput requirements, it is very likely that you will run Weaviate
+on a considerably smaller or larger machine in production.
+
+In [this section below](#what-happens-if-i-run-with-fewer-or-more-cpu-cores-than-on-the-example-test-machine)
```
[the remark this answers is not in the document]
Comment (iamleonie) [reply developers/weaviate/benchmarks/ann.md:228]: Decided to specifically name "Benchmakr FAQs"

**Review by daveatweaviate — APPROVED** [review]
Thanks for the updates @iamleonie, LGTM

**Comment by iamleonie** [comment]
https://update-benchmark-page-proposal--tangerine-buttercream-20c32f.netlify.app/developers/weaviate/benchmarks/ann

**Pull request merged**